scripts/verify-artifact-contents

```diff
@@ -30,7 +30,7 @@ done
 
 # We don't want either slf4j-api or any SLF4J implementation jars in the final plugin artifact
 # since this would lead to a linker error (for api) or an SLF4J error (for impl) in the runtime.
-forbidden_library_prefixes=(checker ide-probe junit-driver lombok scala slf4j)
+forbidden_library_prefixes=(checker ide-probe idea junit-driver lombok scala slf4j)
```

Comment:
Unrel: we don't want any `idea` stuff to land in the plugin zip, that's the entire _idea_ of the plugins that IntelliJ dependencies are provided in the runtime by the IDE ;D